public/app/features/search/state/SearchStateManager.test.ts

```diff
@@ -15,6 +18,8 @@ jest.mock('@grafana/runtime', () => {
 });
 
 describe('SearchStateManager', () => {
+  // jest.spyOn(lodash, 'debounce').mock;
```

Comment:
Why not deleting this instead of keeping it commented out?

Reply:
sorry. this shouldn't have been committed at all 😅   
I'm not sure how this snuck in..........